depends/packages/packages.mk

```diff
@@ -3,7 +3,7 @@ packages:=boost openssl libevent zeromq gmp chia_bls backtrace
 qt_native_packages = native_protobuf
 qt_packages = qrencode protobuf zlib
 
-qt_linux_packages:=qt expat dbus libxcb xcb_proto libXau xproto freetype fontconfig libX11 xextproto libXext xtrans
+qt_linux_packages:=qt expat dbus libxcb xcb_proto libXau xproto freetype fontconfig
```

Comment:
should drop `depends/packages/libX11.mk`, `depends/packages/libXext.mk`, `depends/packages/xextproto.mk` and `depends/packages/xtrans.mk` _files_ too

Reply:
Good point 😄 Done with the last force push.